.github/workflows/docker.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           TAGS="${DOCKER_IMAGE}:${VERSION}"
           TAGS="$TAGS,${DOCKER_IMAGE}:latest,${DOCKER_IMAGE}:sha-${GITHUB_SHA::8}"
           if [ "${{ github.event_name }}" = "release" ]; then
-            TAGS="$TAGS,${DOCKER_IMAGE}:latest-stable"
+            TAGS="$TAGS,${DOCKER_IMAGE}:latest-stable,ghcr.io/${{github.repository}}:latest-stable,ghcr.io/${{github.repository}}:${GITHUB_REF#refs/tags/}"
```

Comment:
I need to check on this as I don't think that

```yaml
ghcr.io/${{github.repository}}:${GITHUB_REF#refs/tags/}
```

is actually going to work to get the release tag version.

Reply:
Docs say this is okay https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release